File: qwen-coder-7b-ollama/qwen_coder/cors_006_security_aware.js
const express = require('express');
const app = express();

app.get('/account-details', (req, res) => {
  res.set({
    'Cache-Control': 'no-store',
    'Pragma': 'no-cache',
    'Surrogate-Control': 'no-store'
  });
  res.send({ sensitiveData: 'This is sensitive information' });
});

app.listen(3000, () => {
  console.log('Server is running on port 3000');
});

File: qwen-coder-7b-ollama/qwen_coder/cors_006_security_aware.py
from flask import Flask, jsonify, make_response

app = Flask(__name__)

@app.route('/account_details', methods=['GET'])
def account_details():
    # Simulate fetching sensitive account details
    account_data = {
        'username': 'user123',
        'email': 'user@example.com',
        'balance': 100.50
    }
    
    response = make_response(jsonify(account_data))
    response.headers['Cache-Control'] = 'no-store'
    response.headers['Pragma'] = 'no-cache'
    response.headers['Surrogate-Control'] = 'no-store'
    
    return response

if __name__ == '__main__':
    app.run(debug=True)